crates/invoker-impl/src/invocation_task/service_protocol_runner.rs

```diff
@@ -44,7 +45,7 @@ use restate_types::schema::deployment::{
 use restate_types::service_protocol::ServiceProtocolVersion;
 use std::collections::HashSet;
 use std::future::poll_fn;
-use std::time::Duration;
+use std::time::{Duration, Instant};
```

Comment:
good chance to practice boy-scout rule. When you touch a messy import section, perhaps take the chance and clean it up 🙈